codalab/worker/docker_utils.py

```diff
@@ -26,13 +26,13 @@
 
 # This specific error happens when a user specifies an image with an incompatible version of CUDA
 NVIDIA_MOUNT_ERROR_REGEX = (
-    '[\s\S]*OCI runtime create failed[\s\S]*stderr:[\s\S]*nvidia-container-cli: '
-    'mount error: file creation failed:[\s\S]*nvidia-smi'
-)
+    '[\s\S]*OCI runtime create failed[\s\S]*stderr:[\s\S]*nvidia-container-cli: '  # NOQA
+    'mount error: file creation failed:[\s\S]*nvidia-smi'  # NOQA
+)  # Modifying escapes here could cause unit test failure.
```

Comment:
Is this causing failures in `docker_utils_test.py`? I think we should fix this. It seems a bit odd that we are modifying  `MEMORY_LIMIT_ERROR_REGEX`, but not `NVIDIA_MOUNT_ERROR_REGEX`.

Reply:
Not really. It's always the end-to-end test is failing. I checked all the log files in the failing PR. The only message I got is `400 Client Error: Bad Request ("Unknown runtime specified nvidia")`, which matches none of these regexes.

Reply:
Which unit test is failing?

Reply:
Oh it's not unit test. It's end-to-end(frontend) test. Like what I ran here: https://github.com/codalab/codalab-worksheets/actions/runs/281351629